### DISCUSSION TITLE
Cucumber integration

### DISCUSSION THREAD

**Discussion by MalteBoehm** [fzyzcjy/flutter_convenient_test#226] [Ideas]
Hey, i would love to see some sort of StepDefinitions in the future. As it is right now we got a lot of code duplication while testing. 

Cucumber/Gherkin with Stepdefinitions would fix that and would make BDD even better.

Here is one implementation:
https://www.npmjs.com/package/cypress-cucumber-preprocessor

**Comment by fzyzcjy** [comment]
> we got a lot of code duplication while testing.

Why? You can just extract functions instead of duplicating code :)

I checked that library. It is like another DSL, which is already very similar to Dart's `test` package's DSL (which our package surely uses).
